src/project/internal/saveprojectscenario.cpp

```diff
@@ -102,14 +102,25 @@ RetVal<io::path> SaveProjectScenario::askLocalPath(INotationProjectPtr project,
 
     io::path defaultPath = configuration()->defaultSavingFilePath(project, filenameAddition);
 
-    QString filter = qtrc("project", "MuseScore File") + " (*.mscz)";
+    QStringList filter;
+    filter << qtrc("project", "Compressed MuseScore file") + " (*.mscz)"
+           << qtrc("project", "Uncompressed MuseScore folder")
+           << qtrc("project", "Uncompressed MuseScore file") + " (*.mscs)";
```

Comment:
Do we really need to expose this mscs format for all users? What is the purpose of this?

Reply:
Now that MSCX is made up of multiple files, I think @igorkorsukov created the MSCS format as a way to have an uncompressed format that includes all data in a single file. This could be useful for storing scores in a Git repository, for example, though it's not an absolute necessity of course given that MSCX can be used for that purpose.

Reply:
Yes, that's what MSCS does. I could imagine that it is better to remove rather MSCX and keep MSCS. Because the MSCX format consists of multiple files in a folder, it is likely to cause users trouble (for example, any extra files that are put in the folder of a MSCX file will be lost when the user re-saves the MSCX file). The MSCS format is just a single file, so that's just safe.

Reply:
I think both formats are imperfect but in different ways. I don't think it is possible to keep everyone satisfied with just one of them.